unit_tests/multiply_two_numbers.py

```diff
@@ -0,0 +1,14 @@
+def multiply_2_nums(num1: int, num2: int) -> int:
+    """
+
+    :param num1: int : Number 1 used in the multiplication
+    :param num2: int: Number 2 used in the multiplication
+    :return: int: The result of the multiplication of num1 and num2
```

Comment:
Use `rtype` to describe return type. Example:

```
:return: The result of the multiplication of num1 and num2.
: rtype: int
```
Also, if the function raises an exception, add it to the docstring.  This [article](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html?highlight=domains#info-field-lists) gives you more information,